### PR TITLE
Check group infrequent combinations

### PR DESCRIPTION
**Pull request by mvankessel-EMC**
1. Added example to documentation
2. Minor code changes
3. Added tests

### CODE REVIEW

File: tests/testthat/test-groupInfrequentCombinations.R [hunk 1]
```diff
@@ -18,3 +18,84 @@ test_that("minimal", {
       groupCombinations = groupCombinations),
     "data.frame")
 })
+
+
+data <- treatment_pathways[[1]][1:5, ]
+
+# Do stuff
```
Comment (rossdwilliams) [inline tests/testthat/test-groupInfrequentCombinations.R:25]: do what stuff...?

**Comment by rossdwilliams** [comment]
@mvankessel-EMC can you fix this merge conflict and add a _slightly_ more useful comment then good to approve

File: tests/testthat/test-groupInfrequentCombinations.R (hunk 1 above)
Comment (mvankessel-EMC) [reply tests/testthat/test-groupInfrequentCombinations.R]: Changed comment

**Comment by mvankessel-EMC** [comment]
> @mvankessel-EMC can you fix this merge conflict and add a _slightly_ more useful comment then good to approve

Done.

**Review by rossdwilliams — APPROVED** [review]

**Pull request merged**